### PR TITLE
[Update] - API and SDK Docs

### PR DESCRIPTION
**Pull request by jaketarnow**
This PR updates all relevant documentation for API endpoints and SDK

### CODE REVIEW

File: documentation/00_overview.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ An index of all pages available in this documentation.
 - [Get Mapping Names](./testnet/public_endpoints/11_get_mapping_names.md)
 - [Get Mapping Value](./testnet/public_endpoints/12_get_mapping_value.md)
 - [Get State Path for Commitment](./testnet/public_endpoints/13_get_state_path_for_commitment.md)
-- [Get Beacons](./testnet/public_endpoints/14_get_beacons.md)
+- [Get Committee](./testnet/public_endpoints/14_get_committee)
```
Comment (christianwwwwwwww) [inline documentation/00_overview.md:81]: append `.md` to this link

File: documentation/00_sdk_overview.md [hunk 2]
```diff
@@ -33,7 +32,7 @@ first zero knowledge web app.
 #### Source: [`sdk/sdk`](https://github.com/AleoHQ/sdk/tree/testnet3/sdk)
 
 
-## 2. [Create-Aleo-App](./sdk/create-aleo-app/00_app_installation.md) - Zero Knowledge Web App Examples
+## 2. [Create-leo-App](./sdk/create-aleo-app/00_app_installation.md) - Zero Knowledge Web App Examples
```
Comment (christianwwwwwwww) [inline documentation/00_sdk_overview.md:35]: "Zero-Knowledge" with hyphen

File: documentation/testnet/getting_started/00_overview.md [hunk 3]
```diff
@@ -81,7 +81,7 @@ docker start -a snarkos
 - [Get Mapping Names](../public_endpoints/11_get_mapping_names.md)
 - [Get Mapping Value](../public_endpoints/12_get_mapping_value.md)
 - [Get State Path for Commitment](../public_endpoints/13_get_state_path_for_commitment.md)
-- [Get Beacons](../public_endpoints/14_get_beacons.md)
+- [Get Beacons](../public_endpoints/14_get_committee)
```
Comment (christianwwwwwwww) [inline documentation/testnet/getting_started/00_overview.md:84]: append `.md` to this link

**Review by christianwwwwwwww — CHANGES_REQUESTED** [review]

**Review by christianwwwwwwww — APPROVED** [review]
LGTM 🤓

**Review by d0cd — APPROVED** [review]

**Pull request merged**